client/tests/acceptance/user-can-click-package-edit-test.js

```diff
@@ -145,19 +145,19 @@ module('Acceptance | user can click package edit', function(hooks) {
 
     await visit('/packages/1/edit');
 
-    assert.dom('[data-test-dcpprojectareaindustrialbusinesszonename]').doesNotExist();
+    assert.dom('[data-test-dcpprojectareaindutrialzonename]').doesNotExist();
     await click('[data-test-dcpprojectareaindustrialbusinesszone="Yes"]');
-    assert.dom('[data-test-dcpprojectareaindustrialbusinesszonename]').exists();
+    assert.dom('[data-test-dcpprojectareaindutrialzonename]').exists();
   });
 
   test('Landmark or Historic District sub Q shows conditionally', async function (assert) {
     this.server.create('project', 1, 'applicant');
 
     await visit('/packages/1/edit');
 
-    assert.dom('[data-test-dcpisprojectarealandmarkname]').doesNotExist();
+    assert.dom('[data-test-dcpProjectarealandmarkname]').doesNotExist();
     await click('[data-test-dcpIsprojectarealandmark="Yes"]');
-    assert.dom('[data-test-dcpisprojectarealandmarkname]').exists();
+    assert.dom('[data-test-dcpProjectarealandmarkname]').exists();
```

Comment:
I don't understand why did the names/selectors change here

Reply:
The test selectors was referring to the property for the related
yes/no question, instead of the text input itself.